jaypeasee/sorting-hat-quiz
# Project setup

#### What’s this PR do?
1. Sets up the file structure
2. Sets up Router for all components with a Switch
3. Makes a mixins and variables file
4. Loosely structures the scss and jsx elements in each component
5. Removes unnecessary inherited React App code and pseudocode

#### Where should the reviewer start?
Review `App.js` to see the full branching tree of components.

#### How should this be manually tested?
1. Run `npm start`
2. Look at `/` route
3. Look at `/your-name` route
4. Look at `/question` route
5. Look at `/result` route
6. Look at any other route to see the Error Component.

#### Any background context you want to provide?
This is the first PR


#### What are the relevant tickets?
* #11 
* #10 
* #7 
* #5 
* #4 
* #3 
* #2
